src/masonite/exceptions/ExceptionHandler.py

```diff
@@ -38,6 +38,9 @@ def handle(self, exception):
                 f"{exception.__class__.__name__}Handler"
             ).handle(exception)
 
+        if getattr(exception, "get_response"):
+            return response.view(exception.get_response())
```

Comment:
I am not sure how it could be done here, because I would like the status code to be controller by the exception.
If I am doing ```return response.view(exception)```, exception.get_response() will be called but status code will be 200.
Here the `AuthorizationException` is implementing get_response which is returning something like `"Not authorized", 403`.

Reply:
Couldn't you do something like 

```python
response.view(exception.get_response(), exception.get_status())
```

Reply:
I'll do that !